binding.gyp

```diff
@@ -72,6 +77,7 @@
     }
   ],
   "variables": {
+    "cpp_version%":"17",
```

Comment:
If tree-sitter has a different default c++ version, please set it here.
You also might need to add another conditional (`node_module_version<131`) to set the cpp version to 17.